spellcheck/platform/platform_spellcheck.h

```diff
@@ -12,9 +12,11 @@ namespace Platform::Spellchecker {
 
 constexpr auto kMaxSuggestions = 5;
 
+[[nodiscard]] bool IsAvailable();
 [[nodiscard]] bool CheckSpelling(const QString &wordToCheck);
 [[nodiscard]] bool IsWordInDictionary(const QString &wordToCheck);
 
+void KnownLanguages(std::vector<QString> *langCodes);
```

Comment:
Why we can't just return `std::vector<QString>`?

Reply:
We can. I use pointer to vector to ensure consistency with FillSuggestionList and CheckSpellingText.